src/main/java/picard/sam/SetNmMdAndUqTags.java

```diff
@@ -73,6 +73,9 @@
     @Option(doc = "Whether the file contains bisulfite sequence (used when calculating the NM tag).")
     public boolean IS_BISULFITE_SEQUENCE = false;
 
+    @Option(doc = "Only set the UQ tag, ignore MD and NM")
+    public boolean SET_ONLY_UQ = false;
```

Comment:
I think that given that there's currently only this use-case, and that changing the code to calculate arbitrary combinations of UQ, MD, and NM is more complicated, I would call this pre-mature complication. if, in the future there's a need for more flexibility, we could add it then.